lib/ansible/plugins/action/win_reboot.py

```diff
@@ -71,6 +72,7 @@ def run(self, tmp=None, task_vars=None):
         connect_timeout_sec = int(self._task.args.get('connect_timeout_sec', self.DEFAULT_CONNECT_TIMEOUT_SEC))
         pre_reboot_delay_sec = int(self._task.args.get('pre_reboot_delay_sec', self.DEFAULT_PRE_REBOOT_DELAY_SEC))
         test_command = self._task.args.get('test_command', self.DEFAULT_TEST_COMMAND)
+        msg = int(self._task.args.get('msg', self.DEFAULT_REBOOT_MESSAGE))
```

Comment:
That's weird, I thought I did that and my branch has it fixed.
I just force-pushed it again.